debug.log

```diff
@@ -0,0 +1,3 @@
+[1201/183509.753:ERROR:directory_reader_win.cc(43)] FindFirstFile: System nie moze odnalezc okreslonej sciezki. (0x3)
```

Comment:
Please add it to the .gitignore

Reply:
It already is in .gitignore.

Reply:
Then you need to remove it and do commit after